### PR TITLE
add new path and params for top and bottom caption

### PR DESCRIPTION
**Pull request by rohangrge**
image

test link-https://sbob-baral.herokuapp.com/q?top=hi&bottom=test

solves issue #3

### CODE REVIEW

File: app/saas.py [hunk 1]
```diff
@@ -16,9 +16,13 @@ def home(*vargs):
     return render_template("display.html", meme_image=filename), 200
 
 
-@app.route("/<path:vargs>", methods=["GET"])
-def meme(vargs):
-    captions = vargs.split('/')
+@app.route("/q", methods=["GET"])
+def meme():
+    top = request.args.get('top')
```
Comment (aditeyabaral) [inline app/saas.py:23]: You will need to handle cases where either the top or the bottom captions aren't provided. You need to also store them in a dictionary to store which caption it is.

Comment (aditeyabaral) [reply app/saas.py:23]: Additionally, these changes will also affect utils.py, in the `generateImage()` function.

**Review by aditeyabaral — CHANGES_REQUESTED** [review]
You need to store the captions in a dictionary, like `data = {"top": top, "bottom": bottom}` and use this dictionary to set the caption type in generateImage in utils.

File: app/saas.py (hunk 1 above)
Comment (rohangrge) [reply app/saas.py:23]: Yeah sure, I'll make the changes and get back

**Comment by rohangrge** [comment]
I have solved everything you asked for,added a fix to caching also ,the images were getting cached on browser end and not updating when requested for a new meme,that is solved now.All other functionalities are working.If no params are given,it falls back to the handleErrorMeme() memes.
test link-https://sbob-baral.herokuapp.com/q?top=hi

File: app/saas.py (hunk 1 above)
Comment (rohangrge) [reply app/saas.py:23]: Solved in new commits

**Review by aditeyabaral — APPROVED** [review]

**Comment by aditeyabaral** [comment]
Looks good, merging.

**Comment by rohangrge** [comment]
Probably need to update the Readme and close the issue

**Comment by aditeyabaral** [comment]
Will do once I test and deploy

**Pull request merged**